index.bs

```diff
@@ -180,88 +181,180 @@ public IP address scope is being discussed in
 
 ## Terminology ## {#req-terminology}
 
-A <dfn>device</dfn> is in the same local network as the user agent (UA), capable
-of HTTPS/WSS server, compliant with [=certificate grant and issuance=] and
-[=certificate management=], and notifies the UA of the capability during
-[=device discovery=].
+A user agent (<dfn>UA</dfn>) is a web browser on a user’s PC, smartphone, tablet
+and so on, which is connected to a local network.
 
-A <dfn>pre-installed certificate</dfn> is a self-signed server certificate
-statically installed in the [=device=].
+A <dfn>device</dfn> is in the same local network as the [=UA=], capable of HTTPS/WSS server.
 
-A <dfn>device CA</dfn> is a certificate authority, and issues a short-term self-signed
-server certificate to a [=device=] authenticated by the UA.
+A <dfn>web service</dfn> is hosted on the internet and whose frontend page is
+loaded on the [=UA=], which accesses to the [=device=] with HTTPS or WSS on the local network.
+
+A <dfn>Web PKI certificate</dfn> is a TLS server certificate that can chain up to
+a root CA (Certificate Authority) trusted by the [=UA=]  (preinstalled on the [=UA=]).
+
+A <dfn>non-Web PKI certificate</dfn> is a TLS server certificate that cannot chain
+up to the root CA, or a self-signed certificate.
```

Comment:
One question: Do we need to add an explicit definition of "root CA" here?

Reply:
I think we don't need to add the definition because it is a general term and it is not used in later sections.